package.json

```diff
@@ -10,7 +10,7 @@
 		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
 		"test": "vitest",
 		"test:once": "vitest run",
-		"test:ci": "vitest run --dir ./test/components && vitest run --dir ./test/lib",
+		"test:ci": "vitest run --dir ./test/unit-and-integration --coverage",
```

Comment:
Can CI not run the other tests?

Reply:
No, the e2e is broken in main and we are still working on it.  Some of the issues were fixed in @mattheworris's PR which I just rebased.